reactNativeProject/src/components/Navigator/index.js

```diff
@@ -0,0 +1,10 @@
+import AppContainer from "../../screens/Main";
+import ScreenOne from "../../screens/ScreenOne";
+import { createBottomTabNavigator } from "react-navigation";
+
+const Tn = createBottomTabNavigator({
```

Comment:
The trello card says:
"Se pide declarar un StackNavigator para que dentro tenga una screen como TabNavigator que tenga la vista de TODOs y una vista dummy la cual va a tener el listado de libros que haremos en la próxima card."
You are missing the `StackNavigator` here

Reply:
Is it right if I do this?

![captura de pantalla 2018-10-16 a la s 17 45 02](https://user-images.githubusercontent.com/42941989/47046330-3d99de00-d16b-11e8-88de-e50ce785ad8b.png)

Reply:
Yes, this is fine.